README.md

```diff
@@ -17,15 +17,25 @@ must be set in the enviromnent before anything will work.
 * `DATABASE_PASSWORD`: The MySQL database password.
 * `GOOGLE_CLIENT_ID`: Google OAuth API client id.
 * `GOOGLE_CLIENT_SECRET`:  Google OAuth API secret.
+* `MODERATOR_FLAVOR`: What flavor of the moderator to run.
```

Comment:
This seems to be about where the comments come from... maybe call it something more like that? 

COMMENT_SOURCE='external-pushes'
vs 
COMMENT_SOURCE='youtube'

I guess in time this may end up being a list? Or perhaps configured via the UI?

Reply:
Originally, I saw this value as a something more general than just "comment source".  We could change labelling in the UI depending on flavour,, (e.g., relabel article as video for youtube deployments.)

But I now think MODERATOR_FLAVOR is a bad idea.  I think I'll just get rid of it, and have the moderator flavour selected in the UI.